content/md/franklin-insights/franklin-insights-subscriptions.md

```diff
@@ -0,0 +1,38 @@
+---
+id: franklin-insights-subscriptions
```

Comment:
I think this article is missing a kind of definition of what is a product subscription. Like what you get by doing this. Maybe give an example based on some screenshots.

Reply:
👍